enclave-protocol/src/lib.rs

```diff
@@ -61,6 +61,7 @@ pub enum IntraEnclaveRequest {
     },
     EndBlock,
     Encrypt(Box<IntraEncryptRequest>),
+    General(String),
```

Comment:
what is "General(String)" supposed to do?

Reply:
it's just for testing,i'll remove it.